lldb/source/Core/Debugger.cpp

```diff
@@ -1402,22 +1402,24 @@ void Debugger::SetDestroyCallback(
 static void PrivateReportProgress(Debugger &debugger, uint64_t progress_id,
                                   std::string title, std::string details,
                                   uint64_t completed, uint64_t total,
-                                  bool is_debugger_specific) {
+                                  bool is_debugger_specific,
+                                  uint32_t is_aggregate) {
```

Comment:
nit: I'd use the enum type here instead of the `uint32_t`
```suggestion
                                  ProgressReportType report_type) {
```